content/docs/querying/operators.md

```diff
@@ -17,7 +23,7 @@ The following binary arithmetic operators exist in Prometheus:
 Binary arithmetic operators are defined between scalar/scalar, vector/scalar,
 and vector/vector value pairs.
 
-**Between two scalars**, the behavior is obvious: they evaluate to another
+**Between two scalars**, the behaviour is obvious: they evaluate to another
```

Comment:
Oh, and btw., our tech writer recommends also not using abbreviations at all, but using "for example", "that is" (or equivalent), etc. Not super urgent though.